src/main/java/org/osiam/client/AuthService.java

```diff
@@ -65,6 +71,7 @@
     private static final String TOKEN_ENDPOINT = "/oauth/token";
     private static final String TOKEN_REVOCATION_ENDPOINT = "/token/revocation";
     private static final String TOKEN_VALIDATION_ENDPOINT = "/token/validation";
+    private static final String CLIENT_ENDPOINT = "/Client";
```

Comment:
Apparently all SCIM endpoints start with a capital letter too, so please ignore the comment.